docs/conversion/sakai_23_oracle_conversion.sql

```diff
@@ -0,0 +1,4 @@
+-- SAK-46974
+ALTER TABLE MFR_MESSAGE_T ADD COLUMN SCHEDULER NUMBER(1,0) DEFAULT NULL;
```

Comment:
ALTER TABLE MFR_MESSAGE_T ADD SCHEDULER NUMBER(1,0) DEFAULT NULL;

Reply:
Is this a boolean value? If so, why null for the default? Select a more appropriate default, either on or off depending on the intention of the feature.

Reply:
Done